Modulefile

```diff
@@ -1,5 +1,5 @@
 name    'ghoneycutt-ntp'
-version '3.3.0'
+version '3.3.1'
```

Comment:
This happened because you did a `git merge` instead of a `git rebase`. This shouldn't be part of the commit history. Luckily, this will not bite us in this situation.